AsyncDisplayKit/ASMapNode.h

```diff
@@ -11,7 +11,10 @@
 
 @interface ASMapNode : ASImageNode
 
-- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
+/**
+ The current region of ASMapNode. This can be set at any time and ASMapNode will animate the change.
+ */
+@property (nonatomic, assign) MKCoordinateRegion region;
```

Comment:
Add to the comment - "this property may be set from a background thread before the node is loaded, and will automatically be applied to define the region of the static snapshot (if liveMap = NO) or the internal MKMapView (otherwise)."